src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
+@SuppressWarnings("unchecked")
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
```

Comment:
Why not constant val but some calculations?

Reply:
As far as I know, bitwise operations improve code performance due to the shift register feature. But, using it in this place is the result of looking back at the original HashMap implementation.